worker-workflow/src/main/java/com/github/cafdataprocessing/workflow/WorkflowManager.java

```diff
@@ -40,19 +43,27 @@
 
 public class WorkflowManager {
 
+    private final static Logger LOG = LoggerFactory.getLogger(WorkflowManager.class);
```

Comment:
Prefer `static final` over `final static` (think science fiction not fiction science)

Reply:
Changed to static final